src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -92,6 +96,25 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
                 c =>
                 {
                     var docOptions = document.GetOptionsAsync(c.CancellationToken).WaitAndGetResult(c.CancellationToken);
+                    var workspaceOptions = document.Project.Solution.Workspace.Options;
+
+                    // AB Test
+                    // If the feature is OFF and the user has not previously been assigned to a test group, assign them now
```

Comment:
Why bother with this? IIRC you can just just check the experimentation service, it will have already decided whether the flight is enabled for this user.